### PR TITLE
Slice params fix for the __init__.py in data folder

### PR DESCRIPTION
**Pull request by bkyryliuk**

### CODE REVIEW

File: tests/core_tests.py [hunk 1]
```diff
@@ -155,8 +156,9 @@ def test_slices_V2(self):
                 (slc.slice_name, 'slice_url', slc.slice_url),
             ]
         for name, method, url in urls:
-            print("[{name}]/[{method}]: {url}".format(**locals()))
+            logging.info("[{name}]/[{method}]: {url}".format(**locals()))
             response = self.client.get(url)
+        self.logout()
```
Comment (ascott) [inline tests/core_tests.py]: ah, forgot to log out that different user. curious that it passed on my branch. thanks for the fix!

File: tests/core_tests.py [hunk 2]
```diff
@@ -155,8 +156,9 @@ def test_slices_V2(self):
                 (slc.slice_name, 'slice_url', slc.slice_url),
             ]
         for name, method, url in urls:
-            print("[{name}]/[{method}]: {url}".format(**locals()))
+            logging.info("[{name}]/[{method}]: {url}".format(**locals()))
```
Comment (ascott) [inline tests/core_tests.py]: why is `logging.info` preferred over `print` here?

Comment (bkyryliuk) [inline tests/core_tests.py]: logging is more flexible and it possible to choose what log severity you care about.
http://stackoverflow.com/questions/6918493/in-python-why-use-logging-instead-of-print

File: tests/core_tests.py (hunk 1 above)
Comment (bkyryliuk) [inline tests/core_tests.py]: it didn't fix it - will keep digging

Comment (ascott) [inline tests/core_tests.py]: i'll take a look too

**Pull request closed without merging**